### PR TITLE
SFDP-49 - Add Notify Status Checking

### PR DESCRIPTION
**Pull request by sfitz42**
- **Create cron job**
- **Add notify status check**

### CODE REVIEW

File: app/data/index.js
Comment (AbidemiAdio) [inline app/data/index.js:1]: Why do we have significant change/update in this code file when compare it to that of standard fcp template?

File: app/repos/notification-log.js [hunk 1]
```diff
@@ -20,4 +20,39 @@ const logRejectedNotification = async (message, notifyError) => {
   })
 }
 
-export { logCreatedNotification, logRejectedNotification }
+const getPendingNotifications = async () => {
+  const pending = await db.notifyApiRequestSuccess.findAll({
```
Comment (AbidemiAdio) [inline app/repos/notification-log.js:24]: I don't see findAll causing any issue now, but may be by the time we have many subscribers with thousands of request, so, we may need to fetch records in batches and not just one go, when fetching all at one go becomes a pain.

Comment (sfitz42) [reply app/repos/notification-log.js:24]: Good point. I suppose it depends on how many requests we're expecting per minute. Might be something for us to find out and define.

File: app/data/index.js
Comment (sfitz42) [reply app/data/index.js:1]: It's a side affect of the decision to use convict for config validation. Not as simple as Joi for the database config as we can't put the hooks / retry directly into the object as we can do with Joi.

Comment (sfitz42) [reply app/data/index.js:1]: Aligned with FCP standard

**Pull request merged**